website/docs/sigs.md

```diff
@@ -64,19 +64,21 @@ class Main
 
   sig do
     params(
-      x: String,      # ← x is a positional param
-      y: String,      # ← y is a keyword param
+      a: String,      # ← required positional param
+      b: String,      # ← optional positional param
+      c: String,      # ← required keyword param
+      d: String,      # ← optional keyword param
```

Comment:
Would you mind using `w` / `x` / `y` / `z` for these variable names to be consistent with our docs style guide:

https://github.com/sorbet/sorbet/blob/master/website/style-guide.md